connector-servicenow/src/main/resources/application.properties

```diff
@@ -1,3 +1,4 @@
+notifications.connector.http.components=https
```

Comment:
Today we can create Http or Https integrations to Splunk and Snow.
If I understand properly, this will restrict connector usage to https only, am I right ?
If yes, it shouldn't ;)

Reply:
`http` URLs are allowed for Splunk and SNow endpoints in `backend`, but there's a check in the connectors that will reject `http` URLs. We need to forbid `http` in `backend`. I'll create a Jira ticket for that.

Reply:
https://issues.redhat.com/browse/RHCLOUD-29412